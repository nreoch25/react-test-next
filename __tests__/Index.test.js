import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import IndexPage from "../pages/index";

describe("Index Page", () => {
  it("matches the snapshot", () => {
    const wrapper = shallow(<IndexPage />);
    const appComponent = wrapper.find("[data-test='component-index-page']");
    expect(appComponent.length).toBe(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
