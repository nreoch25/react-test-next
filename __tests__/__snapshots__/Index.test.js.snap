// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page matches the snapshot 1`] = `
<div
  data-test="component-index-page"
>
  <h1>
    Index Page
  </h1>
</div>
`;
